3 primjeri/ispisivanje_rijeci_for_loop.cpp

```diff
@@ -1 +1,15 @@
+#include <iostream>
 
+int main()
+{
+	int broj = 0;
+
+	std::string rijec;
+	std::cin >> rijec;
+
+	for (rijec[broj]; broj < rijec.size(); broj++) 
```

Comment:
Prvi dio for petlje izvodi kod, koji se dogodi prije drugog dijela.

u svom privom dijelu si napisala rijec[broj];
sto je jednako ovome kodu
`3;`

Napisao sam ti clanak o for petljama, procitaj ga. Mislim da dobro objasnjava 1, 2 i 3ci dio for petlje
https://github.com/Mirixy17/MindPower/wiki/for-loop

Htio bih da prepravis svoj kod tako da ne napises prvi dio for petlje ovako:
`for (; broj < rijec.size(); broj++) `
Posto nista ne radis u prvom dijelu petlje ovo je idealna perpravka

Standard u programiranju je da se tvoja for petlja napise ovako
`for (int broj = 0; broj < rijec.size(); broj++) `

Sada kreiras varijablu broj unutar for petlje(gdje je jedino i koristis)
Posto je varijabla kreirana u for petlji, mozes maknuti kreaciju varijable iz linije 5

Slobodno komentiraj na ovaj komentar ako ti nesto nije jasno.